google/auth/external_account_authorized_user.py

```diff
@@ -222,7 +222,7 @@ def can_refresh(self):
             (self._refresh_token, self._token_url, self._client_id, self._client_secret)
         )
 
-    def get_project_id(self):
+    def get_project_id(self, request=None):
```

Comment:
this parameter is not used, why add it?

Do you mind adding an `Args` and `Returns` comment like the `to_json` method below.

Reply:
gcloud passes it and currently crashes

Reply:
I've added Args and Returns, and used them to explain why we don't use request